sync-android/src/main/java/com/cloudant/sync/sqlite/android/encryption/AttachmentStreamFactory.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) 2015 IBM Cloudant, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
```

Comment:
Test comment on PR diff to see if comments & PRs stay around after deleting a remote branch.